rollup.config.ts

```diff
@@ -21,6 +21,10 @@ const plugins = [
     mainFields: ['module', 'main', 'jsnext:main', 'browser'],
     extensions: ['.js', '.jsx', '.ts', '.tsx'],
   }),
+  replace({
+    preventAssignment: true,
+    'process.env.NODE_ENV': JSON.stringify('development'),
+  }),
```

Comment:
If we are building why we use `development` as NODE_ENV and not `production` ?

Reply:
I consider MSW to be a development tool. Since it never runs in production, developers should get the benefit of warnings from third-party packages visible only in development.

Reply:
ok fine, makes sense :)